src/material-experimental/mdc-slider/slider.spec.ts

```diff
@@ -6,14 +6,337 @@
  * found in the LICENSE file at https://angular.io/license
  */
 
+import {dispatchMouseEvent, dispatchPointerEvent} from '@angular/cdk/testing/private';
+import {Component, DebugElement, Type} from '@angular/core';
+import {ComponentFixture, TestBed} from '@angular/core/testing';
+import {RippleRef, RippleState} from '@angular/material/core';
+import {By} from '@angular/platform-browser';
+import {Thumb} from '@material/slider';
+import {MatSliderModule} from './module';
+import {MatSlider, MatSliderThumb, MatSliderVisualThumb} from './slider';
 
-/* tslint:disable-next-line:no-unused-variable */
-import {MatSlider} from './index';
+describe('MDC-based MatSlider' , () => {
+  beforeAll(() => {
+    // Mock #setPointerCapture as it throws errors on pointerdown without a real pointerId.
+    spyOn(Element.prototype, 'setPointerCapture');
+  });
 
-// TODO(wagnermaciel): Implement this in a separate PR
+  function createComponent<T>(component: Type<T>): ComponentFixture<T> {
+    TestBed.configureTestingModule({
+      imports: [MatSliderModule],
+      declarations: [component],
+    }).compileComponents();
+    return TestBed.createComponent<T>(component);
+  }
 
-describe('MDC-based MatSlider' , () => {
   describe('standard slider', () => {
-    it('does nothing yet', () => {});
+    let fixture: ComponentFixture<StandardSlider>;
+    let sliderDebugElement: DebugElement;
+    let sliderInstance: MatSlider;
+    let inputInstance: MatSliderThumb;
+
+    beforeEach(() => {
+      fixture = createComponent(StandardSlider);
+      fixture.detectChanges();
+      sliderDebugElement = fixture.debugElement.query(By.directive(MatSlider));
+      sliderInstance = sliderDebugElement.componentInstance;
+      inputInstance = sliderInstance._getInput(Thumb.END);
+    });
+
+    beforeEach(done => {
+      fixture.whenStable().then(() => done());
```

Comment:
Is this really necessary? most of our tests don't do this. Have you tried putting `waitForAsync` on the previous `beforeEach` block?